docs/morevp.md

```diff
@@ -351,18 +387,32 @@ It’s possible for an honest user to start an exit and have their exit bond sla
 This can occur if one of the inputs to a transaction is malicious and signs a second transaction spending the same input.
 
 The following scenario demonstrates this attack:
-1. Mallory spends `UTXO1` in `TX1` to Bob, creating `UTXO2`.
+1. Mallory spends `UTXO1m` and Alice spends `UTXO1a` in `TX1` to Bob, creating `UTXO2`.
 2. `TX1` is in-flight.
 3. Operator begins withholding blocks while `TX1` is still in-flight.
-4. Bob starts an exit referencing `TX1` and places `exit bond`.
-5. Mallory spends `UTXO1` in `TX2`.
+4. Alice starts an exit referencing `TX1` and places `exit bond`.
```

Comment:
It's not relevant to the case, but we should mention here that `Alice` also piggybacks to her input and puts `pg-bond`. This bond will be refunded to her - but I think it makes the scenario more clear and logical... Here `Alice` has to piggyback to the input in `period 1` otherwise it will be lost, so the `Mallory` can wait as long as she piggybacks and then reveals the competitor.